scripts/snyk-containers-scan/run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+repo_dir=$GITHUB_WORKSPACE
+file_to_search=Dockerfile
+tmp_dir="${repo_dir}/tmp.${RANDOM}"
+severity_threshold=critical
+docker_workspace=/opt/workspace/
+repo_name="$(basename "$repo_dir")"
+timestamp=$(date +%s)
+stdout_file=${repo_name}.${timestamp}
+pull_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+PR_URL="$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/pulls/$pull_number/files"
+
+mkdir -p $tmp_dir
+
+# # if there is no Dockerfile in this repo, let's no waste time
+# if [[ ! -f "$repo_dir/$file_to_search" ]]; then
+#     echo "This repo appear to not contain any Dockerfile, skipping container security scans"
+#     exit 0
+# fi
+
```

Comment:
I suggest that this script is also ran every time a commit is pushed to `main` branch (i.e. a PR is merged).

To know if the script has been triggered by a "push" event or a "pull request" event you can look for `GITHUB_BASE_REF` variable, if it exists it's a PR, if not it's a push. See an example [here](https://github.com/Typeform/ci-standard-checks/blob/main/scripts/secrets-scan/run.sh#L46-L51).

This way a container image scan will be performed when:
* Any engineer modifies a `Dockerfile`, no matter to which team they belong to
* A PR is merged to `main` branch. If there are vulnerabilities the `CODEOWNERS` will receive an email with the alert (`ci-standard-checks` run failed)